app.js

```diff
@@ -1,27 +1,76 @@
 const productsCenterDOM = document.querySelector(".products-center");
+let cartDOM = document.querySelector(".cart-btn");
+const cart = document.querySelector(".cart");
+const cartOverlay = document.querySelector(".cart-overlay");
+let closeCartDOM = document.querySelector(".close-cart");
 
 // get all products
-const products = [];
 
-let results = "";
-for (let index = 0; index < products.length; index++) {
-  results += `
-    <article class="product">
-        <div class="img-container">
-            <img
-                src="${products[index].url}"
-                alt="product"
-                class="product-img"
-            />
-            <button class="bag-btn" data-id="1">
-                <i class="fas fa-shopping-cart"></i>
-                add to bag
-            </button>
-        </div>
-        <h3>${products[index].name}</h3>
-        <h4>$${products[index].price}</h4>
-    </article>
-`;
+const getProducts = async function () {
```

Comment:
why do you need this code? You just need to focus on showing and hiding cart by clicking on the icon.

Reply:
Thanks @imambux for your honest review , i have fixed the concerns.